lib/compiler/scjson-to-module.js

```diff
@@ -455,14 +511,17 @@ var actionTags = {
             targetVariableName = '_scionTargetRef',
             targetDeclaration = 'var ' + targetVariableName + ' = ' + target + ';\n';
 
-        var event = 
-        ["{", 
-         "   target: " + targetVariableName + ",", 
-         "   name: " + processAttr(action, 'event') + ",", 
-         "   type: " + processAttr(action, 'type') + ",", 
-         "   data: \n" + constructSendEventData(action) + ",", 
-         "   origin: _sessionid", 
-         "}"].map(function(line){return '     ' + line;}).join('\n');   //lightweight formatting
+        var event =
+        "{\n" +
+         "  target: " + targetVariableName + ",\n" +
+         "  headers: " + processAttr(action, '{http://www.247-inc.com/scxml}headers') + ",\n" +
+         "  enctype: " + processAttr(action, '{http://www.247-inc.com/scxml}enctype') + ",\n" +
+         "  fetchtimeout: " + processAttr(action, '{http://www.247-inc.com/scxml}fetchtimeout') + ",\n" +
```

Comment:
Yes, we've been hoping for https://github.com/jbeard4/SCION/wiki/IO-Processor-Plugin-API-Specification-for-SCION-2.0-branch#array-customattributes so we can get rid of this and eliminate our fork.